.coveragerc

```diff
@@ -15,28 +15,18 @@ omit =
     homeassistant/components/air_quality/nilu.py
     homeassistant/components/air_quality/norway_air.py
     homeassistant/components/air_quality/opensensemap.py
+    homeassistant/components/air_quality/nilu.py
```

Comment:
The access mode has been changed on `.coveragerc`. That should be reverted.

There are also non related changes here, probably due to not rebasing properly.